.changeset/config.json

```diff
@@ -9,9 +9,11 @@
   "commit": false,
   "fixed": [
     [
+      "@commercetools-applications/*",
       "@commercetools-backend/*",
       "@commercetools-frontend/*",
-      "!@commercetools-frontend/codemod"
```

Comment:
I thought about this again and I think it's better to keep it in sync with the other versions. It would be easier also to document new codemods for specific releases.